src/pages/Library.tsx

```diff
@@ -157,15 +227,316 @@ class Library extends React.Component<ILibraryProps, ILibraryState>
             scrollStamp: e.timeStamp
         });
     }
-    //Change the chapter shown to new chapter of curNum.
-    setChapter(chapter: any){
-        console.log(this.state.context);
+
+    
+    //Track scroll to bottom
+    onIonScroll=  (ev : any) => {
+        const elem = document.querySelector("ion-content")!.getScrollElement();
+        // the ion content has its own associated scrollElement
+        //elem > scrollHeight = 9090 by default.
+        var show = false;
```

Comment:
You don't have to change this, but I recommend using the keyword let to avoid scoping problems in loops.